classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
Ah, sounds like [event sourcing](http://martinfowler.com/eaaDev/EventSourcing.html)? Would that require a substantial data refactor? #gleefullyofftopic